src/components/Table/hooks/useTableLogic.js

```diff
@@ -1,28 +1,29 @@
-import { useState} from 'react';
+// useTableLogic.js
+import { useState, useMemo } from 'react';
 
-const useTableLogic = (initialData) => {
-  const data=initialData
-  const [order, setOrder] = useState("asc");
-  const [orderBy, setOrderBy] = useState("title");
+const useTableLogic = (initialData, setData, setIsModalOpen, setSelectedItemId) => {
+  const [order, setOrder] = useState('asc');
```

Comment:
**For example, values for sorting are better stored in constants**
```
----
export const DIRERECTION_ASC = 'asc';
export const DIRERECTION_DESC = 'desc'
---
import { DIRERECTION_ASC } from 'bla-bla';
const [order, setOrder] = useState(DIRERECTION_ASC);
----
```
**It's also better to pass the value**
` const [orderBy, setOrderBy] = useState('title'); ->  const [orderBy, setOrderBy] = useState(props.initOrderBy);`